### DISCUSSION TITLE
Tutorials and demo site!

### DISCUSSION THREAD

**Discussion by chrisK824** [tomasvotava/fastapi-sso#150] [Show and tell]
Hello all,

I've been using that library for some projects and I have created some, hopefully useful to any of you, resources: 

- Repository to showcase fastapi implementation with minimum frontend flow: https://github.com/chrisK824/fastapi-sso-example
- Demo website: https://fastapi-sso-example.vercel.app/ . This one is using the code from the above repo and apps created by me in each provider.

Full tutorials for many providers:

1. Facebook: https://itnext.io/fastapi-facebook-single-sign-on-sso-b10865535029?sk=61535a516d624fd8c210ce54a99609f6
2. X (Twitter): https://medium.com/itnext/fastapi-x-twitter-single-sign-on-sso-5725671ad180?sk=f8cb0bc186ce9739c7c489c1f58d11a7
3. LinkedIn: https://itnext.io/fastapi-linkedin-single-sign-on-sso-b1a0118b02d7?sk=fbc5ca8da2c74823d7ca9d4be1830235
4. Spotify: https://itnext.io/fastapi-spotify-single-sign-on-sso-4f461b95344c?sk=80c02ea8e23d1db14e2880ec4222799a
5. GitHub: https://itnext.io/fastapi-github-single-sign-on-sso-f6b942395649?sk=8abd525bc3686fa6802f6b824a7e8485
6. GitLab: https://levelup.gitconnected.com/fastapi-gitlab-single-sign-on-sso-624fe442ec56
7. Google: https://itnext.io/fastapi-google-single-sign-on-sso-47454e2e2859?sk=990e5357f2b4cda533f696b5d5682904
8. Microsoft: https://itnext.io/fastapi-microsoft-single-sign-on-sso-d0a0ab248c36?sk=1500649a11028c24b3d95d1c2aa588ea

**Comment by tomasvotava** [comment]
Hi @chrisK824,
thanks again for doing this, it looks awesome and I believe it will help a lot of users to overcome some inital hiccups when starting using either fastapi or fastapi-sso! Kudos! 🎉

**Comment by sajanv88** [comment]
Love it @chrisK824  I am using it on my opensource multi-tenancy app  https://github.com/sajanv88/full_stack_fastapi_react_template/issues/117
